packages/api/src/archive/controller/get_featured.test.ts

```diff
@@ -34,7 +34,9 @@ describe("getFeatured", () => {
 	test("should retrieve featured archives", async () => {
 		const result = await agent.get("/api/v2/archive/featured").expect(200);
 		expect(result.body).toHaveLength(1);
-		const archive = (result.body as FeaturedArchive[])[0];
```

Comment:
I think this whole test block would be much cleaner as:

```
expect(result.body).toMatchObject([{
  archiveId: 3,
  name: "Jay Rando",
  ...
}]);
```

This also prevents the awkward type cast!  (though maybe I'm writing this way too soon and you address this since I suspect that's the "unsafe type assertions in tests" you were talking about)

OK disregard this comment probably!